src/form-field/directives/drf-form-field.js

```diff
@@ -14,26 +17,31 @@
                 replace: true,
                 transclude: true,
                 template: function (elem, attrs) {
-                    var templateBase = 'templates/drf/';
-                    var defaultTemplate = 'drf-form-field.html';
+                    var templateBase = 'templates/drf/form-field/';
+                    var defaultTemplate = 'form-field.html';
                     var templateUrl = templateBase + (angular.isDefined(attrs.type) ? attrs.type + '.html' : defaultTemplate);
                     return $templateCache.get(templateUrl) || $templateCache.get(templateBase + defaultTemplate);
                 },
-                link: function (scope, element, attrs) {
+                link: function (scope, element, attrs, ctrl, transcludeFn) {
                     // Define a default object, so that assigning
                     // scope.label to scope.field.label does not fall over
                     // if scope.field is undefined whilst waiting for an
                     // API response. We'll also need to watch that property on the
                     // scope for changes and update it.
-                    scope.field = scope.$eval(attrs.formField) || {};
+                    scope.field = scope.$eval(attrs.drfFormField) || {};
 
-                    scope.$watch(attrs.formField, function (field) {
-                        scope.field = scope.$eval(attrs.formField) || {};
+                    scope.$watch(attrs.drfFormField, function (field) {
+                        scope.field = scope.$eval(attrs.drfFormField) || {};
                     });
 
                     scope.$watch('field', function (field) {
                         scope.label = angular.isDefined(attrs.label) ? attrs.label : scope.field.label;
                     });
+
+                    transcludeFn(function (clone) {
+                        var transcludeContainer = angular.element(element[0].querySelector('.transclude-container'));
+                        transcludeContainer.replaceWith(clone);
```

Comment:
Because @hippogriffic was complaining about the extra `<div>` that `ng-transclude` introduces. I actually explained that in the commit description.